### PR TITLE
Documentation/readme

### PR DESCRIPTION
**Pull request by lcombs15**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# *imhungry* Android app
+Let *[imhungry](http://www.imhungry.io/)* take the guess work out of deciding what to eat!
```
Comment (TripleFrequency) [inline README.md]: You should use `https` in the link, and drop the `www`.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# *imhungry* Android app
```
Comment (TripleFrequency) [inline README.md]: I don't believe we should say Android app in the headline directly, but could mention it in the body.
Perhaps something like "*imhungry* - Your Mealtime Companion," or something more directly related to the theme of the app.

See: https://github.com/UweTrottmann/SeriesGuide

If we edit the tagline of the repo to mention the platform, then I don't believe we should directly have it in the header of the readme.

File: README.md (hunk 1 above)
Comment (lcombs15) [reply README.md]: `https://imhungry.io/` is a dead link.

Comment (TripleFrequency) [reply README.md]: They all are right now, but `https://imhungry.io` will be the one we end up using. The current "live" one is just what namecheap threw on it by default.

**Pull request merged**